plc4j/integrations/apache-nifi/nifi-plc4x-processors/src/main/java/org/apache/plc4x/nifi/util/Plc4xCommon.java

```diff
@@ -0,0 +1,149 @@
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcList) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcLREAL) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcLTIME) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcLWORD) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcNull) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcREAL) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().doubleType().endUnion().noDefault();		
+			}else if (entry.getValue() instanceof PlcSINT) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().intType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcSTRING) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcStruct) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcTIME_OF_DAY) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcTIME) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcUDINT) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcUINT) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcULINT) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcUSINT) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcWCHAR) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();
+			}else if (entry.getValue() instanceof PlcWORD) {
+				builder.name(fieldName).type().unionOf().nullBuilder().endNull().and().stringType().endUnion().noDefault();				
+			}
```

Comment:
@chrisdutz ?
if we aren't remove this commented code, replace with a comment that we do not yet handles these in the javadoc

Reply:
still issue